readme.md

```diff
@@ -1,88 +1,183 @@
-# FingerprintJS
-
-Work in progress, stay tuned.
+<p align="center">
+  <a href="https://fingerprintjs.com">
+    <img src="resources/logo.svg" alt="FingerprintJS" width="300px" />
+  </a>
+</p>
+<p align="center">
+  Browser fingerprinting solution
+</p>
+<p align="center">
+  <a href="https://github.com/fingerprintjs/fingerprintjs/actions?workflow=Test">
+    <img src="https://github.com/fingerprintjs/fingerprintjs/workflows/Test/badge.svg" alt="Build status">
+  </a>
+  <a href="https://www.npmjs.com/package/@fingerprintjs/fingerprintjs">
+    <img src="https://img.shields.io/npm/dt/fingerprintjs2.svg" alt="Total downloads from NPM">
+  </a>
+  <a href="https://www.npmjs.com/package/@fingerprintjs/fingerprintjs">
+    <img src="https://img.shields.io/npm/v/@fingerprintjs/fingerprintjs.svg" alt="Current NPM version">
+  </a>
+</p>
 
 ## Quick start
 
-### In browser
+### Install from CDN
 
 ```html
 <script>
-  function onFingerprintJSLoad(fp) {
-    fp.get().then(({ visitorId }) => console.log(visitorId));
+  function onFingerprintJSLoad(fpAgent) {
+    // The FingerprintJS agent is ready. Get a visitor identifier when you'd like to.
+    fpAgent.get().then(result => {
+      // This is the visitor identifier:
+      const visitorId = result.visitorId;
+      console.log(visitorId);
+    });
   }
 </script>
 <script
-  async src="https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3.0.0-beta.2/dist/fp.min.js"
+  async src="https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.min.js"
   onload="FingerprintJS.load().then(onFingerprintJSLoad)"
 ></script>
 ```
 
-A simple way to start but not recommended because AdBlock and similar browser extensions often block this URL.
-You should at least upload the script to your server.
+This is a simple way to start but not recommended because AdBlock and other browser extensions can block the script.
+You should at least upload the script file to your server.
 
-### Webpack/Rollup/Browserify
+### Or from NPM and use with Webpack/Rollup/Browserify
```

Comment:
Install from NPM to use with ..

Reply:
I want “or” to show that this is an alternative way to install, not the second step of installation

Reply:
then maybe "Alternatively you can install from NPM to use with..."?

Reply:
LGTM

Reply:
ok thanks, please update other minor wording changes and let's merge into v3

Reply:
Sure, I'm going to do it on Monday